### DISCUSSION TITLE
Query regarding grain size and domain size

### DISCUSSION THREAD

**Discussion by sushdash** [neperfepx/neper#310] [Q&A] [answered]
Hello!
As a beginner, I am confused about the way neper considers the grain size and domain size. I mean to begin any deformation, it is necessary to understand the length unit but I find it really difficult to grasp over here. Can you please explain this a bit?

**Comment by rquey** [comment, marked as answer]
Neper uses no length unit.

- The default domain size is 1 x 1 x 1 and you can consider this meters, millimeters, etc.
- The typical input is the number of grains (option `-n`). In that case, the average grain size will depend on the domain size and the number of grains.
- Another input is via the grain size, using `-n from_morpho -morpho "diameq:lognormal(mean,standard_deviation)"`, where `mean` is the average grain size. Again, you can consider this meters, millimeters, etc., but, of course, it has to be consistent with the domain size. In that case, the number of grains will depend on the domain size and the average grain size.

**Reply by sushdash** [reply]
Thank you for your explanation. 
I just have one more query that when we are using the Neper output for FEpX simulations, how to consider the length unit?

**Reply by rquey** [reply]
Similarly, there is no length scale effect in FEPX, so, an FEPX model can be in meters, millimeters, etc.
